templates/include/glean/histogram.1.parquetmr.txt

```diff
@@ -0,0 +1,14 @@
+optional group range (TUPLE) {
```

Comment:
Should the histogram type require any of these?

Reply:
The JSON schema doesn't currently require any of these, but I feel like that's just an oversight on my part.  Definitely `range`, `bucket_count`, and `values` should be required.  `histogram_type` has an obvious default (linear) so could be optional, and `underflow` and `overflow` could default to zero.  Or I'd be fine with everything required, too.

Reply:
Great, I'll update that schema as well.